src/Shared/Recruit.Vacancies.Client/Infrastructure/Repositories/MongoDbVacancyRepository.cs

```diff
@@ -207,15 +207,15 @@ public async Task<IEnumerable<long>> GetAllVacancyReferencesAsync()
         public async Task<IEnumerable<ProviderVacancySummary>> GetVacanciesAssociatedToProvider(long ukprn)
         {
             var builder = Builders<Vacancy>.Filter;
-            var filter = builder.Eq(ProviderUkprnFieldName, ukprn) &
-                        builder.Ne(IsDeletedFieldName, true);
-                
+            var filter = 
+                builder.Eq(ProviderUkprnFieldName, ukprn) &
+                builder.Ne(IsDeletedFieldName, true);
```

Comment:
Since we are not persisting nulls, default behavior of Boolean is  going to be false. This is a positive check, in the sense that we are looking for where value is true (which is explicitly set) and then filtering it out. But it would work either way.